modules/tinymce/src/core/main/ts/api/dom/ScriptLoader.ts

```diff
@@ -170,10 +173,10 @@ class ScriptLoader {
    */
   public add(url: string, success?: () => void, scope?: any, failure?: () => void) {
     const state = this.states[url];
+    this.queue.push(url);
```

Comment:
This was needed, as without this the success/failure callbacks aren't handled in `loadScripts` if the script had already been loaded (which was also a bug). This then meant the `scriptLoadedCallbacks` wasn't cleaned up because of the circular scope reference.